pkg/e2e/compose_run_test.go

```diff
@@ -37,7 +37,7 @@ func TestLocalComposeRun(t *testing.T) {
 			"Hello one more time")
 		lines = Lines(res.Stdout())
 		assert.Equal(t, lines[len(lines)-1], "Hello one more time", res.Stdout())
-		assert.Assert(t, !strings.Contains(res.Combined(), "orphan"))
+		assert.Assert(t, strings.Contains(res.Combined(), "orphan"))
```

Comment:
question: was this a bug? :o

Reply:
more or less. We only considered "orphans" for service container not declared in current compose model. This PR also consider stopped `run` containers as orphaned